package.json

```diff
@@ -22,11 +22,11 @@
   },
   "homepage": "https://github.com/ruby/setup-ruby",
   "dependencies": {
-    "@actions/cache": "^1.0.5",
-    "@actions/core": "^1.2.6",
-    "@actions/exec": "^1.0.3",
-    "@actions/io": "^1.0.2",
-    "@actions/tool-cache": "^1.3.4"
+    "@actions/cache": "^1.0.7",
+    "@actions/core": "^1.4.0",
+    "@actions/exec": "^1.1.0",
+    "@actions/io": "^1.1.1",
+    "@actions/tool-cache": "^1.7.1"
```

Comment:
Could you update dependencies in a separate PR? That way we could already merge that and also more easily find out if that causes some issue.

Reply:
> also more easily find out if that causes some issue.

Passed?  Ok...

Reply:
See PR #195